docsets/Poco/docset.json

```diff
@@ -1,14 +1,18 @@
 {
     "name": "Poco",
-    "version": "1.9.0",
+    "version": "1.9.4",
     "archive": "Poco.tgz",
```

Comment:
Note: You need to update the archive at this location as well. I'll use `versions/1.9.4/Poco.tgz` this time.